app/controllers/admin/shelters_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::SheltersController < ApplicationController 
+  def index 
+    @shelters = Shelter.all
+  end
+
+  def show
+    @shelter = Shelter.find(params[:id])
+    #wondering which method to put these ^^ in for admin.
```

Comment:
There's no need to create an additional admin/shelter model as this isn't a new database, but rather a different page to view the same shelter data. `admin` is a namespace, which is really just a tool for organizing routes (in this case, distinguishing the applicant-facing routes from the admin-facing routes) rather than relating to a specific database resource. We'll learn more about namespacing in the Advanced Routing class after break.